.circleci/config.yml

```diff
@@ -153,7 +153,7 @@ jobs:
       - run:
           name: PHP CS
           command: |
-            vendor/bin/phpcs -d memory_limit=512M -p -n --extensions=php --standard=PSR2 ./src ./public/index.php && vendor/bin/phpcs -d memory_limit=512M -p --extensions=php --standard=vendor/glpi-project/coding-standard/GlpiStandard/ --ignore=/.git,/config,/files,/lib,/node_modules,/plugins,/src,/tests/config,/vendor ./
+            vendor/bin/phpcs -d memory_limit=512M -p -n --extensions=php --standard=PSR2 ./src ./public/index.php && vendor/bin/phpcs -d memory_limit=512M -p -n --extensions=php --standard=vendor/glpi-project/coding-standard/GlpiStandard/ --ignore=/.git/,/config/,/files/,/lib/,/node_modules/,/plugins/,/src/,/tests/config/,/vendor/ ./inc
```

Comment:
I'm OK with this change, but not to check `inc` only. Several other directories must be checked as well.

Also, this issues do exist on 9.4, doesn't it?

Reply:
Trailing slashes are present on 9.4 branch on elements that were a problem here, this is why I targetted master branch.

And inclusion of inc is a mistake, well catch.

Reply:
Ok :+1: then :)